lib/web_console/middleware.rb

```diff
@@ -113,5 +115,13 @@ def respond_with_unavailable_session(id)
 
         Rack::Response.new(body, status, headers).finish
       end
+
+      def respond_with_unacceptable_request
+        status = 406
+        headers = { 'Content-Type' => 'application/json; charset = utf-8' }
+        body    = { output: '406 Not Acceptable' }.to_json
```

Comment:
Can we change the format here? At the moment, we return exactly what the status tells us. :-) We may return something like `{ "error": "A supported version is expected in the Accept header.` }`.